src/gfx/scene.rs

```diff
@@ -258,7 +270,9 @@ impl Scene {
         self.modelview = mat4_to_uniform(value);
     }
 
-    pub fn set_lights<F>(&mut self, writer: F) where F: FnOnce(&mut [f32]) {
+    pub fn set_lights<F>(&mut self, writer: F)
+        where F: FnOnce(&mut [f32])
+    {
```

Comment:
Ooh, this is contentious, I always want my braces on the preceeding line!  I thought rustfmt agreed, or did that change recently?